modules/promotions/promotion-data.php

```diff
@@ -0,0 +1,58 @@
+<?php
+namespace Elementor\Modules\Promotions;
```

Comment:
Not sure if this is the correct place. This `data` folder is used to extend the WordPress Rest API (for adding new routes).

Maybe you can add this in `elementor/includes/api.php`?

Reply:
@louiswol94 Is see that the `elementor/includes/api.php` is used specifically for the library data, and it might be complicated to make it handle different requests.

Could we create a class in here: `elementor/includes/api-requests/editor-assets-api.php` or something like that?

Reply:
@hein-obox I see it is communicating with different endpoints like INFO, FEEDBACK, so just add ASSETS? WDYT?

```
$api_info_url
$api_feedback_url 
```

Reply:
@louiswol94 I think the will be need to do quite a bit of refactoring.

Reply:
@hein-obox Okay, maybe then just move the file to `elementor/includes/editor-assets-api.php` for now.

Reply:
@louiswol94 Done